staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -37,7 +37,7 @@ type InTreePlugin interface {
 	// TranslateInTreeInlineVolumeToCSI takes a inline volume and will translate
 	// the in-tree inline volume source to a CSIPersistentVolumeSource
 	// A PV object containing the CSIPersistentVolumeSource in it's spec is returned
```

Comment:
Add comments podNamespace is only needed for azurefile to fetch secret namespace. No need to be set for other plugins.

Reply:
fixed, thanks.